src/bundle/Controller/RoleController.php

```diff
@@ -79,7 +80,16 @@ public function listAction(): Response
     {
         $roles = $this->roleService->loadRoles();
 
+        $rolesNumbers = array_column($roles, 'id');
+
+        $roleBulkRemoveData = new RoleBulkRemoveData(
```

Comment:
Why do you need another data class instead of refactoring `RoleDeleteData` to accept multiple roles? This would reduce code duplication as you could share this class between removing single role via Role View page and removing multiple roles in the Role List page.